### DISCUSSION TITLE
Headings in wikis are incorrectly being styled as hyperlinks

### DISCUSSION THREAD

**Discussion by CritLoren** [community/community#69367] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

# Issue description:
Hello, I noticed a bug in one of my repos that I could reproduce on Vivaldi and Edge, namely the heading styles seem to be broken (it uses the hyperlink style instead colour and text decoration wise). This makes navigating around wikis confusing when headings are indistinguishable from hyperlinks, considering some heading sections might actually use hyperlinks as well. Navigating from a repo's home to its wiki first (without refreshing) seems to be the only way to avoid this issue, but any hyperlinks to /wiki pages are currently showing the broken version.

# Reproduction steps:
1. Go to any repo with a wiki, just to the repo's home page (for example https://github.com/CritLoren/Yggdrasil-VR/)
2. Navigate to the Wiki tab
3. Refresh the page
4. See the heading style break

The bug also happens just by going directly to a wiki page link, but this way you can notice the difference.

# Some examples:
### Broken style:
![image](https://github.com/community/community/assets/22298459/ef9e8f90-7ca4-4de9-a76f-2d23c8072c2d)
### Correct style:
![vivaldi_esXxWZnXQF](https://github.com/community/community/assets/22298459/6f866c43-5c0e-49d2-9be4-9bc4d03fee86)

### Broken style:
![vivaldi_UjUY5QmaJ8](https://github.com/community/community/assets/22298459/e8d3187b-3190-4edd-a34b-4de334c9f32a)
### Correct style:
![vivaldi_Vv4VgokIYW](https://github.com/community/community/assets/22298459/ef2c44f0-4d2c-4fe6-9300-bc6bc2463cdd)

### Broken style:
![vivaldi_J5SmxwGMc4](https://github.com/community/community/assets/22298459/0c781700-775b-4003-9078-e42bd42ef146)
### Correct style:
![vivaldi_aJpk9De1BO](https://github.com/community/community/assets/22298459/bb57f4c3-3799-4dc2-8f2f-0224924a4f32)

**Comment by ghostinhershell** [comment]
Hey @CritLoren, thanks so much for reaching out about this. It looks like this issue was caused by a recent update to underline links so that they could be easily distinguishable from surrounding text. You can learn more about that update here: [Underline Links in Text Blocks Public Beta](https://github.com/orgs/community/discussions/68734). 

The issue that you were seeing where headings were being underlined as well has now been resolved, per @dipree's [comment here](https://github.com/orgs/community/discussions/68734#discussioncomment-7218640). Take a look and let me know if you're still seeing this issue on your end.

**Reply by CritLoren** [reply]
Hello the issue seems to still be present when I test it. Going on a repo that has a wiki from it's code tab to its wiki tab vs opening the same URL of the wiki section in a new tab still causes the issue to appear. I've cleared all data for the GitHub URL via the developer tools when checking.

**Reply by dipree** [reply]
Thanks for reporting this. We are looking into it.

**Reply by Timmichi** [reply, marked as answer]
@CritLoren It looks like this was recently fixed. I'm no longer seeing headers link to themselves as hyperlinks. @dipree Was there an announcement of the fix?
